ApplicationCatalogGui/1.0.0/installer-manifest.json

```diff
@@ -0,0 +1,17 @@
+{
+    "wire-navigation": [
+        {
+            "navigations": {
+                "application-catalog-gui": {
+                    "label": "Apps",
+                    "title": "Apps",
+                    "icon": "fa-archive",
+                    "bundle": "application-catalog-gui",
+                    "controller": "index",
+                    "action": "index"
+                }
+            },
+            "after": "users"
```

Comment:
What if this entity doesn't exist in the customer's project menu?

Reply:
The new entry will then be added to the end of the navigation.